website/docs/r/privatelink_endpoint_serverless.html.markdown

```diff
@@ -0,0 +1,64 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: privatelink_endpoint_serverless"
+sidebar_current: "docs-mongodbatlas-datasource-privatelink-endpoint-serverless"
+description: |-
+Describes a Serverless PrivateLink Endpoint
+---
+
+
+# Resource: privatelink_endpoint_serverless
+
+`privatelink_endpoint_serverless` Provides a Serverless PrivateLink Endpoint resource.
+This is the first of two resources required to configure PrivateLink for Serverless, the second is [mongodbatlas_privatelink_endpoint_service_serverless](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/privatelink_endpoint_service_serverless).
+
+-> **NOTE:** Groups and projects are synonymous terms. You may find group_id in the official documentation.
+-> **NOTE:** Feature unavailable in Free and Shared-Tier Clusters
+This feature is not available for M0 free clusters, M2, and M5 clusters. To learn more about which features are unavailable, see [Atlas M0 (Free Cluster), M2, and M5 Limitations.](https://www.mongodb.com/docs/atlas/reference/free-shared-limitations/#std-label-atlas-free-tier)
+## Example Usage
+
+### Basic
+```terraform
+
+resource "mongodbatlas_privatelink_endpoint_serverless" "test" {
+	project_id   = "<PROJECT_ID>"
+	instance_name = mongodbatlas_serverless_instance.test.name
+	provider_name = "AWS"
+}
+	  
+resource "mongodbatlas_serverless_instance" "test" {
+	project_id   = "<PROJECT_ID>"
+	name         = "test-db"
+	provider_settings_backing_provider_name = "AWS"
+	provider_settings_provider_name = "SERVERLESS"
+	provider_settings_region_name = "US_EAST_1"
+	continuous_backup_enabled = true
+}
+```
+
+
+## Argument Reference
+
+* `project_id` - (Required) Unique 24-digit hexadecimal string that identifies the project.
+* `instance_name` - (Required) Human-readable label that identifies the serverless instance.
+* `provider_name` - (Required) Cloud provider name; AWS is currently supported
+
```

Comment:
Since you added timeouts don't we need something like https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/privatelink_endpoint_service#timeouts?